airflow/utils/cli.py

```diff
@@ -53,51 +54,57 @@ def _check_cli_args(args):
         )
 
 
-def action_logging(f: T) -> T:
-    """
-    Decorates function to execute function at the same time submitting action_logging
-    but in CLI context. It will call action logger callbacks twice,
-    one for pre-execution and the other one for post-execution.
-
-    Action logger will be called with below keyword parameters:
-        sub_command : name of sub-command
-        start_datetime : start datetime instance by utc
-        end_datetime : end datetime instance by utc
-        full_command : full command line arguments
-        user : current user
-        log : airflow.models.log.Log ORM instance
-        dag_id : dag id (optional)
-        task_id : task_id (optional)
-        execution_date : execution date (optional)
-        error : exception instance if there's an exception
-
-    :param f: function instance
-    :return: wrapped function
-    """
-
-    @functools.wraps(f)
-    def wrapper(*args, **kwargs):
+def action_cli(check_db=True):
```

Comment:
Ah, this is why we have `action_cli()` instead of `action_cli` at usage. See https://stackoverflow.com/a/69210737/439189



```suggestion
def action_cli(func=None, check_db=True):
```

Reply:
Nice!